app/js/components/manageApps/AddAddon.jsx

```diff
@@ -1,11 +1,3 @@
-/* * This Source Code Form is subject to the terms of the Mozilla Public License,
- * v. 2.0. If a copy of the MPL was not distributed with this file, You can
- * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
```

Comment:
Did you intentionally remove this license header?

Reply:
No

Reply:
So can you add them back?